pkg/db/db_test.go

```diff
@@ -0,0 +1,435 @@
+package db_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/theredditbandit/pman/pkg/db"
+	bolt "go.etcd.io/bbolt"
+)
+
+func Test_GetDBLoc(t *testing.T) {
+	t.Run("Test getDBLoc", func(t *testing.T) {
+		expectedWords := []string{".local", "share", "pman"}
+
+		actualPath, err := db.GetDBLoc(db.DBTestName)
+		defer os.Remove(actualPath)
+
+		require.Equal(t, err, nil)
+		require.Contains(t, actualPath, expectedWords[0], expectedWords[1], expectedWords[2], db.DBTestName)
```

Comment:
Contains doesn't work like this

github.com/stretchr/testify/require

You should have something like

```go
require.Contains(t, actualPath, expectedWords)
require.Contains(t, actualPath, db.DBTestName)
```